README.md

```diff
@@ -92,8 +92,9 @@ Now for some actual coding. In the `bcrypt-exercise` folder, you will find every
 
 __Steps:__
 1. Have a quick read of the `bcryptjs` documentation [here](https://www.npmjs.com/package/bcryptjs). `bcryptjs` is a pure JavaScript implementation of `bcrypt` (as opposed to the JavaScript/C++ [`bcrypt`](https://www.npmjs.com/package/bcrypt) npm package).
-2. `$ npm install`
-3. In `index.js`, implement the functions `hashPassword`, and `comparePasswords`, so that the tests in `index.test.js` pass. `hashPassword` accepts a `String`, and should asynchronously return a `String`. `comparePasswords` accepts two `String`s and returns a `Boolean`, also asynchronously.
+2. `cd into the `bcrypt-exercise folder
```

Comment:
I think your backticks are a bit messed up. It would be better to match the format of the install command too
```suggestion
2. `$ cd bcrypt-exercise`
```